README.md

```diff
@@ -59,14 +73,13 @@ dsm_agent_download_hostname | String | The hostname of the Deep Security Manager
 dsm_agent_download_port | Int | The port to connect to the Deep Security Manager to download the agents. This is typically the same port as the one used to access the Deep Security Manager administration interface. | 443
 dsm_agent_activation_hostname | String | The hostname for the agents to communicate with once deployed. For Marketplace and software deployments, this is typically the same hostname as 'dsm_agent_download_hostname'. | agents.deepsecurity.trendmicro.com
 dsm_agent_activation_port | Int | The port used for the agent heartbeat (the regular communication). For Marketplace and software deployments, the default is 4120. | 443
-tenant_id | String | In a multi-tenant installation (like Deep Security as a Service), this identifies the tenant account to register the agent with. |
-token/tenant_password | String | In a multi-tenant installation (like Deep Security as a Service), this identifies the tenant account to register the agent with. In latest Deep Security Manager, "tenant_password" has been replaced with "token". "tenant_password" has been kept for backwards compatibility |
-policy_id | String | The Deep Security ID assigned to the policy and applied to the agents on activation. |
+tenant_id | String | In a multi-tenant installation (like Deep Security as a Service), this identifies the tenant account to register the agent with. | Only required in multi-tenancy environment
```

Comment:
The requirement is not default value, should be merged as part of description.
If you'd like to high-light this, maybe we can use BOLD style.